docs/spec-files/empty.md

```diff
@@ -4,13 +4,13 @@ Sometimes, only headers need to be communicated; there is no content for the bod
 
 The [PIURI](#protocol-identifier-uri) for this protocol is:
 
-    https://didcomm.org/empty/1.0
+    https://didcomm.org/empty/2.0
 
 The `empty` message has no semantic meaning. The message's only purpose is to allow the transfer of message headers. 
 
 ```JSON
 {
-  "type": "https://didcomm.org/empty/1.0/empty",
+  "type": "https://didcomm.org/empty/2.0/empty",
```

Comment:
same as comment above. This didn't need a version rev.

Reply:
My comment is also the same as above.  :-)